src/Layout/LandingPage/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Redirect } from 'react-router-dom'
+import { Title, LandingImg, Paragraph, Button, ResourceLink, Link } from './index.style';
+import landing from '../../assets/landing-img.jpeg';
+import Header from '../../CommonComponents/Header';
```

Comment:
We should only call the header component in the app because it shows in almost all of the pages

Reply:
@95Rawan  
Done!